src/OnlineSales/Entities/TaskExecutionLog.cs

```diff
@@ -0,0 +1,27 @@
+﻿// <copyright file="TaskExecutionLog.cs" company="WavePoint Co. Ltd.">
+// Licensed under the MIT license. See LICENSE file in the samples root for full license information.
+// </copyright>
+
+namespace OnlineSales.Entities
+{
+    public enum TaskExecutionStatus
+    {
+        PENDING = 0,
+        COMPLETED = 1,
+    }
+
+    public class TaskExecutionLog : BaseEntity
+    {
+        public string? TaskName { get; set; }
+
+        public DateTime ScheduledExecutionTime { get; set; }
+
+        public DateTime ActualExecutionTime { get; set; }
+
+        public TaskExecutionStatus Status { get; set; }
+
+        public int RetryCount { get; set; }
+
+        public string? Comment { get; set; }
```

Comment:
Hi @gsplakmal, what is the Comment? I am thinking if it is a good name for this property... What kind of data do you plan to store there? Errors only?